addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/discovery/ZoneDiscoveryService.java

```diff
@@ -90,12 +90,10 @@ public void publishZones(DeviceInformationState state, ThingUID bridgeUid) {
             Map<String, Object> properties = new HashMap<>();
             properties.put(CONFIG_ZONE, zoneName);
 
-            DiscoveryResult discoveryResult = DiscoveryResultBuilder.create(uid)
-                    .withProperties(properties)
-                    .withLabel(state.name + " " + zoneName)
-                    .withBridge(bridgeUid)
-                    .build();
-            
+            DiscoveryResult discoveryResult = DiscoveryResultBuilder.create(uid).withProperties(properties)
+                    .withLabel(state.name + " " + zoneName).withBridge(bridgeUid)
+                    .withRepresentationProperty(CONFIG_ZONE).build();
```

Comment:
Can you also add the representation property to the `zone` thing type definition? See also the [ESH documentation](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#representation-property).